bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingConfigAction.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.binding;
+
+import org.eclipse.smarthome.core.thing.Thing;
+
+
+/**
+ * This class defines the action configuration interface of Eclipse SmartHome
+ *
+ * @author Chris Jackson - Initial contribution and API
+ */
+public interface ThingConfigAction {
```

Comment:
Probably:) I'd need to check (and test!)... The point of this PR was to get some feedback as the discussion in the forum had stalled, so I appreciate the feedback...

I'm in the position where I will need to get this functionality implemented quite soon so if actions are not a concept that people like I'll need to look for alternatives, but I'm not sure how to get similar functionality with the current system...